oh_staff_ui/models.py

```diff
@@ -116,6 +116,17 @@ class ItemLanguageUsage(models.Model):
         Language, on_delete=models.PROTECT, blank=False, null=False
     )
 
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(
+                fields=[
+                    "item",
+                    "value",
```

Comment:
Minor nitpick: remove the trailing comma after "value", and on line 126 after the constraint name, to be consistent with the other new constraints (e.g., see "name_value_type_unique" on line 170).  Or, change all the other new constraints to match this one, but for these short lists I prefer the compact format.